wiki/cpp/DEVELOPER_GUIDE.md

```diff
@@ -267,32 +267,32 @@ TODO: Add this
 To enable `libcuml.so` users to control how memory for temporary data is allocated, allocate device memory using the allocator provided:
 ```cpp
 template<typename T>
-void foo(const ML::cumlHandle_impl& h, cudaStream_t stream, ... )
+void foo(const raft::handle_t& h, cudaStream_t stream, ... )
 {
-    T* temp_h = h.getDeviceAllocator()->allocate(n*sizeof(T), stream);
+    T* temp_h = h.get_device_allocator()->allocate(n*sizeof(T), stream);
     ...
-    h.getDeviceAllocator()->deallocate(temp_h, n*sizeof(T), stream);
+    h.get_device_allocator()->deallocate(temp_h, n*sizeof(T), stream);
 }
 ```
 The same rule applies to larger amounts of host heap memory:
 ```cpp
 template<typename T>
-void foo(const ML::cumlHandle_impl& h, cudaStream_t stream, ... )
+void foo(const raft::handle_t& h, cudaStream_t stream, ... )
 {
-    T* temp_h = h.getHostAllocator()->allocate(n*sizeof(T), stream);
+    T* temp_h = h.get_host_allocator()->allocate(n*sizeof(T), stream);
     ...
-    h.getHostAllocator()->deallocate(temp_h, n*sizeof(T), stream);
+    h.get_host_allocator()->deallocate(temp_h, n*sizeof(T), stream);
 }
 ```
 Small host memory heap allocations, e.g. as internally done by STL containers, are fine, e.g. an `std::vector` managing only a handful of integers.
 Both the Host and the Device Allocators might allow asynchronous stream ordered allocation and deallocation. This can provide significant performance benefits so a stream always needs to be specified when allocating or deallocating (see [Asynchronous operations and stream ordering](#asynchronous-operations-and-stream-ordering)). `ML::deviceAllocator` returns pinned device memory on the current device, while `ML::hostAllocator` returns host memory. A user of cuML can write customized allocators and pass them into cuML. If a cuML user does not provide custom allocators default allocators will be used. For `ML::deviceAllocator` the default is to use `cudaMalloc`/`cudaFree`. For `ML::hostAllocator` the default is to use `cudaMallocHost`/`cudaFreeHost`.
 There are two simple container classes compatible with the allocator interface `MLCommon::device_buffer` available in `ml-prims/src/common/device_buffer.hpp` and `MLCommon::host_buffer` available in `ml-prims/src/common/host_buffer.hpp`. These allow to follow the [RAII idiom](https://en.wikipedia.org/wiki/Resource_acquisition_is_initialization) to avoid resources leaks and enable exception safe code. These containers also allow asynchronous allocation and deallocation using the `resize` and `release` member functions:
```

Comment:
Not really effected by this PR, but noticed this should probably be updated anyways.

```suggestion
There are two simple container classes compatible with the allocator interface `MLCommon::device_buffer` available in `src_prims/common/device_buffer.hpp` and `MLCommon::host_buffer` available in `src_prims/common/host_buffer.hpp`. These allow to follow the [RAII idiom](https://en.wikipedia.org/wiki/Resource_acquisition_is_initialization) to avoid resources leaks and enable exception safe code. These containers also allow asynchronous allocation and deallocation using the `resize` and `release` member functions:
```